enterprise_access/apps/events/utils.py

```diff
@@ -0,0 +1,147 @@
+"""
+Util classes and methods for producing license events to the event bus. Likely
+temporary.
+"""
+
+import logging
+
+from confluent_kafka import KafkaError, KafkaException, SerializingProducer
+from confluent_kafka.admin import AdminClient, NewTopic
+from confluent_kafka.error import ValueSerializationError
+from confluent_kafka.serialization import StringSerializer
+from django.conf import settings
+
+from enterprise_access.apps.events.data import CouponCodeRequestEvent, CouponCodeRequestEventSerializer
+
+logger = logging.getLogger(__name__)
+
+
+# def generate_coupon_code_request_data(coupon_code_request):
```

Comment:
Can we either remove this block comment or add a meta-comment explaining why it has been left in?